azurerm/internal/services/web/resource_arm_function_app.go

```diff
@@ -225,6 +227,36 @@ func resourceArmFunctionApp() *schema.Resource {
 							Optional: true,
 							Default:  false,
 						},
+						"ip_restriction": {
+							Type:       schema.TypeList,
```

Comment:
Should this be a TypeSet to prevent duplicates & order won't matter here?

Reply:
I am trying to change to a `TypeSet` but it is not working as intended. In my case it will take some time to resolve.

Reply:
It shoul just be a matter of changing how you cast the objects, if you grant me push permissions i'll happily make the change

Reply:
I have granted push permission, you can freely add commits to this Pull Request. I tried #5319 for a similar change, but was investigating because the plan no longer worked.